docs/user-guide/walkthrough/README.md

```diff
@@ -12,77 +12,176 @@ certainly want the docs that go with that version.</h1>
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-# Kubernetes 101 - Walkthrough
+# Kubernetes 101 - Kubectl CLI & Pods
+
+For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple containers
+
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
+- [Kubectl CLI](#kubectl-cli)
+  - [Install Kubectl](#install-kubectl)
+  - [Configure Kubectl](#configure-kubectl)
+- [Pods](#pods)
+  - [Pod Definition](#pod-definition)
+  - [Pod Management](#pod-management)
+  - [Volumes](#volumes)
+  - [Multiple Containers](#multiple-containers)
+- [What's Next?](#whats-next)
+
+
+## Kubectl CLI
```

Comment:
Would be good to fold this and content from https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/gce.md#installing-the-kubernetes-command-line-tools-on-your-workstation into https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/prereqs.md.

But that could be a separate PR.

cc @caesarxuchao